src/dialogs/examples/basic/ListCardDialog.ts

```diff
@@ -0,0 +1,52 @@
+import * as builderteams from "botbuilder-teams";
+import * as builder from "botbuilder";
+import * as config from "config";
+import { TriggerActionDialog } from "../../../utils/TriggerActionDialog";
+import { DialogIds } from "../../../utils/DialogIds";
+import { DialogMatches } from "../../../utils/DialogMatches";
+// import { ListCard, ListCardItem, ListCardItemType } from "../basic/ListCard";
+import { Strings } from "../../../locale/locale";
+
+export class ListCardDialog extends TriggerActionDialog {
+
+    private static async step1(session: builder.Session, args?: any | builder.IDialogResult<any>, next?: (args?: builder.IDialogResult<any>) => void): Promise<void> {
+        let list = Array<builderteams.ListCardItem>();
+
+        // list item1
+        let item1 = new builderteams.ListCardItem();
+        item1.type(builderteams.ListCardItemType.resultItem);
+        item1.icon(config.get("app.baseUri") + "/assets/computer_people.jpg");
+        item1.title(session.gettext(Strings.list_card_item1_title));
+        item1.subtitle(session.gettext(Strings.list_card_item1_subtitle));
+
+        // list item2
+        let item2 = new builderteams.ListCardItem();
+        item2.type(builderteams.ListCardItemType.resultItem);
+        item2.icon(config.get("app.baseUri") + "/assets/computer_person.jpg");
+        item2.title(session.gettext(Strings.list_card_item2_title));
+        item2.subtitle(session.gettext(Strings.list_card_item1_subtitle));
+
+        // we have added only two items for this sample, list card support multiple items as needed
+        list.push(item1);
+        list.push(item2);
+
+        let card = new builderteams.ListCard();
```

Comment:
Here too, use session and chain the fluent methods.
new ListCard(session)
    .title(Strings.list_card_title)
    .addItem(xxx)
....